common4j/src/test/com/microsoft/identity/common/java/nativeauth/providers/NativeAuthRequestHandlerTest.kt

```diff
@@ -148,6 +151,22 @@ class NativeAuthRequestHandlerTest {
         )
     }
 
+    @Test
+    fun testSignUpStartWithUnsetCorrelationIdShouldHaveNilHeader() {
```

Comment:
`testSignUpStartWithUnsetCorrelationIdShouldHaveNilHeader` -> 'testSignUpStartWithUnsetCorrelationIdShouldNotHaveHeader'

Reply:
The title is accurate as it is - the test passes a "UNSET" value, and in this case the header should be nil.

Reply:
@rmccahill can you elaborate? Why should we be sending an empty header (i.e. the `REQUEST_ID` header is set, but it doesn't contain a value), rather than not set the header at all?

Reply:
Ah I see what you mean - in my mind 'nil header' refers to the header not being present. But I can see how that's confusing. I've updated the naming on these tests to reflect the suggested name.